src/pages/Login.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react'
-import { Link, Redirect } from 'react-router-dom'
+import { Link, Navigate } from 'react-router-dom'
```

Comment:
I was getting a lint error from npm that wouldn't even let me start the app because of something like `"undefined export for react-router-dom"`. That means it's saying there's no exported module/function/component/variable called `Redirect` in the `react-router-dom` package.

This is because from v5 to v6, they changed from using the `Redirect` component, and now instead want us to use the `Navigate` component: [https://reactrouter.com/en/main/upgrading/v5#remove-redirects-inside-switch](https://reactrouter.com/en/main/upgrading/v5#remove-redirects-inside-switch)

Reply:
Yes ,exactly this error I had as well, I have to refresh my React to v6, honestly I didn't know that Redirect component has been removed in React Router v6.  Now I will remember that Navigate component was introduced to handle redirects.